src/Components/Carousel/Carousel.css

```diff
@@ -3,15 +3,44 @@
   display: flex;
   width: 100%;
   height: 100%;
+  align-content: center;
 }
 
 .main-img{
   width: 100%;
-  max-width: 750px;
-  max-height: 950px;
+  max-width: 400px;
+  min-width: 150px;
   height: auto;
-  align-content: center;
   z-index: 1;
+  margin-top: 30px;
+}
+
+@media only screen and (min-width: 1231px) {
```

Comment:
Why 1231? If there's a reason for that value (is it a standard?) put it in comments.